src/tasks/selectors.js

```diff
@@ -13,6 +15,17 @@ export function buildFilter(auth, type, value) {
   return filters[type](auth, value);
 }
 
+export function _labelsPool(state) {
```

Comment:
rather not use `_` prefix, can confuse with lodash

Reply:
can easily be arrow function:
```JS
export const labelsPool = ({ tasks }) => tasks.labelsPool;
```

Reply:
Oh god this is so pretty I'm melting here 🥇